packages/contracts-bedrock/scripts/checks/hotfuzz/main.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os/exec"
+	"regexp"
+	"strings"
+	"sync"
+	"time"
+)
+
+// TestInfo stores information about each test
+type TestInfo struct {
+	Name     string
+	Contract string
+}
+
+func main() {
+	timeout := flag.Duration("timeout", 2*time.Second, "timeout duration for each test")
```

Comment:
I think my ideal version of this would accept the following flags:

- `--max-seconds <seconds>`
- `--max-runs <runs>`
- `--match-path <paths>`

Can just apply `--match-path` to `getTestNames`

Reply:
If you supply `--max-runs` and `--max-seconds` then you'll always get the lower of the two